src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -56,11 +59,33 @@
       ;; Respond to all CSR validation failures with a 400
       (middleware-utils/plain-response 400 msg))))
 
-(defn handle-get-certificate-revocation-list
-  [{:keys [cacrl]}]
-  (-> (ca/get-certificate-revocation-list cacrl)
-      (rr/response)
-      (rr/content-type "text/plain")))
+  (schema/defn format-http-date :- (schema/maybe org.joda.time.DateTime)
+    "Formats an http-date into joda time.  Returns nil for malformed or nil http-dates"
+    [http-date :- (schema/maybe schema/Str)]
+    (if http-date
```

Comment:
If you don't want to return a value for the else block, or the else is `nil` you can just use the `when` macro instead of `if`.